boards/arm/nucleo_h563zi/nucleo_h563zi-common.dtsi

```diff
@@ -72,11 +72,22 @@
 	status = "okay";
 };
 
+&i2c1 {
+	pinctrl-0 = <&i2c1_scl_pb8 &i2c1_sda_pb9>;
+	pinctrl-names = "default";
+	status = "okay";
+	eeprom1: eeprom@56 {
```

Comment:
I cannot find any eeprom specified here:
https://www.st.com/en/evaluation-tools/nucleo-h563zi.html#st_all-features_sec-nav-tab
It is added in the overlay as well.